.ci/toolchain-vs2017-amd64-facebook.yml

```diff
@@ -14,6 +14,19 @@ trigger:
     include:
       - .ci/azure-pipelines-aws-agent.yml
       - .ci/templates/toolchain.yml
+schedules:
+- cron: "0 */3 * * *"
```

Comment:
Can we not do something like: '*/90 * * * *'?

Reply:
I don't think that's a valid cron syntax. If you Google for "cron 90 minutes" you will find always solutions with 2 entries.

Reply:
Boo!